### PR TITLE
PR-Agent Pro Models

### PR DESCRIPTION
**Pull request by mrT23**
### **PR Type**
Documentation

___

### **Description**
- Added a new document `PR_agent_pro_models.md` to describe the default and alternative models used by PR-Agent Pro.
- Provided configuration examples for changing the model to Claude-3.5-sonnet.
- Updated the usage guide index to include a link to the new PR-Agent Pro Models documentation.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>PR_agent_pro_models.md</strong><dd><code>Add documentation for PR-Agent Pro models and configuration</code></dd></summary>
<hr>

docs/docs/usage-guide/PR_agent_pro_models.md

<li>Added a new document for PR-Agent Pro models.<br> <li> Described default models (GPT-4) and alternative model <br>(Claude-3.5-sonnet).<br> <li> Provided configuration examples for changing models.<br>

</details>

  </td>
  <td><a href="https://github.com/Codium-ai/pr-agent/pull/1046/files#diff-4418b0a6515176a867f7fc398f0e6ecde9f7b113a8367a62d90adc5f94b2cf05">+21/-0</a>&nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>index.md</strong><dd><code>Update index to include PR-Agent Pro Models link</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

docs/docs/usage-guide/index.md

- Added a link to the new PR-Agent Pro Models documentation.

</details>

  </td>
  <td><a href="https://github.com/Codium-ai/pr-agent/pull/1046/files#diff-dabd9ad2b5e65ed825e46f5ff0060386b51b62a0eedf4ca541aacee2b12af762">+2/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: docs/docs/usage-guide/PR_agent_pro_models.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+```
+
+You can also use different models for different tools. For example, you can use the Claude-3.5-sonnet model only for the `improve` tool (and keep the default GPT-4 model for the other tools) by adding the following to your configuration file:
+```
+[github_app]
+pr_commands = [
+    "/describe --pr_description.final_update_message=false",
+    "/review --pr_reviewer.num_code_suggestions=0",
+    "/improve --config.model=claude-3-5-sonnet",
+]
```
Comment (mrT23) [inline docs/docs/usage-guide/PR_agent_pro_models.md:20]:
```suggestion
[github_app]
pr_commands = [
    "/describe --pr_description.final_update_message=false",
    "/review --pr_reviewer.num_code_suggestions=0",
    "/improve --config.model=claude-3.5-sonnet",
]
```

**Comment by mrT23** [comment]
## Generating PR code suggestions


Work in progress ...<br>

**Comment by mrT23** [comment]
## Generating PR code suggestions


Work in progress ...<br>

**Comment by mrT23** [comment]
## Generating PR code suggestions


Work in progress ...<br>

**Comment by mrT23** [comment]
## Generating PR code suggestions


Work in progress ...<br>

**Comment by mrT23** [comment]
## Generating PR code suggestions


Work in progress ...<br>

**Comment by mrT23** [comment]
## Generating PR code suggestions


Work in progress ...<br>

**Comment by mrT23** [comment]
## Generating PR code suggestions


Work in progress ...<br>

**Comment by mrT23** [comment]
## Generating PR code suggestions


Work in progress ...<br>

**Comment by mrT23** [comment]
## Generating PR code suggestions


Work in progress ...<br>

**Comment by mrT23** [comment]
## Generating PR code suggestions


Work in progress ...<br>

**Review by hussam789 — APPROVED** [review]

**Pull request merged**